system/configuration.nix

```diff
@@ -59,7 +56,7 @@
   # services.xserver.libinput.enable = true;
 
   # Define a user account. Don't forget to set a password with ‘passwd’.
-  users.users.${username} = {
+  users.users.axol = {
```

Comment:
If we're doing this, we should remember to also update the README to fix the reference

This is better for not assuming the single-user case as much, possibly slightly worse if people are running commands without properly reading what they're doing but I guess it's probably fine...